Documentation/joss/paper.md

```diff
@@ -19,50 +9,85 @@ authors:
     orcid: 0000-0003-0387-3440
     affiliation: 1
   - name: Helge Hecht
-    affiliation: 2
+    affiliation: 1
   - name: Julian Amann
     affiliation: 1
   - name: Jonas Schlenger
+    orcid: 0000-0002-4875-5117
     affiliation: 1
 affiliations:
  - name: Chair for Computational Modelling and Simulation, Technical University of Munich
    index: 1
- - name: ??
-   index: 2
 date: 24 December 2020
 bibliography: paper.bib
 ---
 
 # Summary
 
-> TODO: A summary describing the high-level functionality and purpose of the software for a diverse, *non-specialist* audience.
+*TUM Open Infra Platform* (OIP) is an open source application for
+ viewing and analysis of different BIM models used in the civil engineering field.
```

Comment:
Explain what BIM is beforehand -> give a motivation e.g: With Building Information Modeling (BIM) (Eastman et al., 2008; Borrmann et al.,
2015), the technological foundations, tools and methods have been created to further advance the digitization of construction. In this context, BIM describes the idea of integrating all information relevant to the life cycle that is associated with a
relevant to a building model, such as a tunnel, bridge or road, in a digital building model.
and to use this information throughout all phases of the life cycle. This not only includes data on
real components such as walls, piers or abutments, but also abstract objects like
but also abstract objects such as processes, actors or schedules.
schedules. The building model itself is used by various multidisciplinary users such as architects, civil engineers, building owners, public authorities, insurance companies or facility management,
insurance companies or facility management for their respective planning and
planning and utilization goals and, if necessary, specifically adapted and extended.
and extended. Among other things, the digital building model should ensure that

Reply:
This passage is really good. You've cut it off at the end, care to send me the full one? I'll include it in the summary.

Is it from your PhD? So that I cite it correctly.

Reply:
yes its from my PhD 
The digital building model is to ensure, among other things,
- that all relevant data are available to all project participants,
- that all data are in a consistent state (data integrity should be
be guaranteed),
- that the data can be used efficiently

See page 6 -> http://mediatum.ub.tum.de/doc/1453871/1453871.pdf  (search for  "alle für den Lebenszyklus relevanten Informationen")

Reply:
Since the paper should not be that long, I've included a condensed version.